src/DOM/HTML/Indexed/LengthLiteral.purs

```diff
@@ -0,0 +1,12 @@
+module DOM.HTML.Indexed.LengthLiteral where
+
+import Prelude (show, (<>))
+
+data LengthLiteral
+  = Pixels Int
```

Comment:
Because while it is indeed deprecated, and they should probably be integers, old tags must be `String` for compatibility. I don't know how to resolve that other than overloading via typeclass.